app/lib/features/search/widgets/quick_jump.dart

```diff
@@ -76,8 +90,6 @@ class QuickJump extends ConsumerWidget {
                     icon: SvgPicture.asset(
                       'assets/images/tasks.svg',
                       semanticsLabel: 'tasks',
-                      width: 32,
-                      height: 32,
```

Comment:
that is too small now. Please revert his and the `iconSize: 48` above.
![grafik](https://github.com/acterglobal/a3/assets/40496/a51b1a76-34f7-4f00-aa34-a08071210f71)